Scripts/assignment/emme_assignment.py

```diff
@@ -245,6 +254,68 @@ def _create_attributes(self, scenario, attributes):
                 scenario = scenario)
             log.debug("Created attr {} for scen {}".format(extr.name, scenario.id))
 
+    def calc_noise(self):
+        noise_areas = pandas.Series(0, zone_param.area_aggregation)
+        network = self.day_scenario.get_network()
+        morning_network = self.assignment_periods[0].emme_scenario.get_network()
+        for link in network.links():
+            # Aggregate traffic
+            light_modes = ("@car_work", "@car_leisure", "@van")
+            traffic = sum([link[mode] for mode in light_modes])
+            rlink = link.reverse_link
+            if rlink is None:
+                reverse_traffic = 0
+            else:
+                reverse_traffic = sum([rlink[mode] for mode in light_modes])
+            cross_traffic = 0.85 * 0.9 * (traffic+reverse_traffic)
```

Comment:
`calc_noise()` contains several contants, like 0.85 and 0.9 here, and their purposes are unclear for me. :) Consider making them variables with descriptive names ie. `variable_name_telling_what_the_number_means = 0.85` or adding some other comment. Apart from this row, magic numbers appear also in many other lines in this function.

Reply:
These (0.85 and 0.9) were clear to me, so I made the suggested changes. However, other parameters are truly "magic" in the sense that I do not know the meaning of them. I am not sure how to handle them.

Reply:
Good changes! Hmm, the second part is tricky! Is it possible to add a comment pointing to the source of those numbers (like a hyperlink if the source is a report or a website) even if the source does not say anything specific of the numbers? At least, there would be a clue about how to investigate them later. This is optional!